bin/precice-config-visualizer

```diff
@@ -66,9 +66,19 @@ def getParticipantColor(solverinterface, blackOnly=False):
 
 def configToGraph(ast, args):
     assert(ast.tag == "precice-configuration")
+
     solverinterfaces = ast.findall("solver-interface")
-    assert(len(solverinterfaces) == 1)
-    si = solverinterfaces[0]
+    precice_version = -1
+
+    if(len(solverinterfaces) == 1):
+        precice_version = 2
+    elif(len(solverinterfaces) == 0):
+        precice_version = 3
+
+    if(precice_version == 2):
+        root = solverinterfaces[0]
+    elif(precice_version == 3):
+        root = ast
```

Comment:
The compatibility question is basically handled here. @uekerman do you think we need some additional comments to make this more understandable? I think the code is fairly readable.

Reply:
This answers my question: It does support v2 and v3.
We could probably also make this explicit in the documentation. Maybe some users wonder as well.